english/bite-the-marshmallow/ricky/ricky.md

```diff
@@ -3,6 +3,6 @@ He peers into the rear view mirror, head tilt down,
 his eyes rising above his cool-cat sunglasses.
 
 "You don't need to know where we are going. 
-All you need to know is that I'm ..."
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.